lib/openshift/quota_limits.rb

```diff
@@ -18,6 +19,10 @@ def storage_requests_raw
       @storage_requests
     end
 
+    def cpu_raw
```

Comment:
@yapei I think `cup` and `cpu_raw` can be consolidated into just `cpu`

Reply:
got it, let me update

Reply:
updated, could you help review again ? @pruan-rht